crossplane/how-to-guides/use-pre-existing-crossplane-tap.hbs.md

```diff
@@ -0,0 +1,41 @@
+# Use a pre-existing or out-of-band installation of Crossplane with Tanzu Application Platform
+
+This topic describes options for installing Tanzu Application Platform to a cluster which already has an installation of Crossplane, or for which you wish to install Crossplane using an out-of-band mechanism (e.g. through Helm install).
+
+## About
+
+The crossplane.tanzu.vmware.com Package is included in Tanzu Application Platform's Full, Iterate and Run profiles. Any installation of Tanzu Appliation Platform using one of these profiles will result in the installation of Crossplane to the cluster via the Package. In most cases this is desirable behaviour as Crossplane is required to support many of the services capabilities offered by Tanzu Application Platform.
+
+However there are cases where this is not desirable, for example if you are planning to install Tanzu Application Platform to a cluster which already has an installation of Crossplane on it, or if you would like to install Crossplane via some other, out-of-band mechanism (e.g. through Helm install). This guide documents the configuration options available to you for such cases.
+
+## Option 1: Exclude the crossplane.tanzu.vmware.com Package in tap-values.yaml file
+
+The first option is to simply exclude the crossplane.tanzu.vmware.com Package in tap-values.yaml.
+
+```yaml
+# tap-values.yaml
+
+excluded_packages:
+- crossplane.tanzu.vmware.com
+```
+
+Choose this option if you already have Crossplane installed on your cluster, you wish to continue to manage your Crossplane installation out-of-band, and you are either installing or upgrading to Tanzu Application Platform >= v1.5.0.
+
+This option has the effect of completely disabling the crossplane.tanzu.vmware.com Package from the Tanzu Application Platform install, which leaves you free to manage the Crossplane installation in whatever way you see fit.
+
+It is important to note that other Tanzu Application Platform components depend on the availability of Crossplane. One such example is [Bitnami Services](../../bitnami-services/about.hbs.md). Therefore if you are planning to exclude the crossplane.tanzu.vmware.com Package, it is recommended to first install Crossplane using whatever mechanism you choose prior to installing Tanzu Application Platform with the Package excluded.
+
+## Option 2: Adopt existing Crossplane resources
```

Comment:
I wonder if we should actually document this option so prominently. There might be certain configurations of Crossplane with existing managed resources that might be changed during adoption in ways that we haven't foreseen and hence might cause issues for the user. We haven't tested this in-depth and I'm not even sure we can reliably do that considering that there are essentially unlimited configuration permutations for Crossplane.

Reply:
At the very least we might want to add some sort of warning.

Reply:
Hmm fair point. I think we probably need _some_ documentation given it will show up in the list of package values and users can configure it ... perhaps I'll just add it as more of a small note to the end of the doc and also mention that milage may vary ...

Reply:
Actually I think it's better to remove mention of this option all together. Feels a bit weird to document "here is an option but we don't recommend it" ...